datapackage_pipelines_measure/processors/add_uuid.py

```diff
@@ -0,0 +1,28 @@
+import uuid
```

Comment:
The main problem is that every time the pipeline runs, the uuid will be different.
I think a better id might be something more consistent (i.e. coming from the data itself and not random).

What do you think?